File: archived/tests/transpiler/x/fs/query_sum_select.fs
// Generated 2025-07-21 18:37 +0700

let nums: int list = [1; 2; 3]
let result = [ for n in nums do if n > 1 then yield Seq.sum n ]
printfn "%s" (("[" + (String.concat ", " (List.map string result))) + "]")

File: archived/tests/transpiler/x/pas/query_sum_select.pas
{$mode objfpc}
program Main;
var
  nums: array of integer;
  result: array of ;
  n: integer;
begin
  nums := [1, 2, 3];
  result := [];
  for n in nums do begin
  if n > 1 then begin
  result := concat(result, [sum(n)]);
end;
end;
  writeln(result);
end.
